app/javascript/packs/admin/users/editUserModals.js

```diff
@@ -0,0 +1,185 @@
+import { WINDOW_MODAL_ID, showWindowModal } from '@utilities/showModal';
+
+const getModalContent = () => document.getElementById(WINDOW_MODAL_ID);
+
+/**
+ * Populate the add organization modal with user data
```

Comment:
I couldn't decide if the JSDoc comments here were useful or not (given the param names are fairly self-explanatory). I thought it might be helpful to have the summary up top of expected `dataset` attributes, but if folks feel it's too verbose then I'm totally happy to remove 🙂

Reply:
I don't think this is too verbose--I can still scroll the through the file and read through the code without feeling lost in code and comments!